quick-protocol.html

```diff
@@ -463,20 +463,20 @@ <h3 property="schema:name">Overview</h3>
 
                   <p>A notification channel allows a Notification Sender to send notifications to a Notifications Receiver. A notification channel builds upon one of many established web protocols to exchange data, determining its <em>type</em>.</p>
 
-                  <p>The following diagram illustrates the flow of data for those notification channel types in which a client establishes a subscription and then maintains a persistent connection to a notifications source are illustrated by the following diagram.</p>
+                  <p>The following diagram illustrates the flow of data for those notification channel types in which a client establishes a subscription and then maintains a persistent connection to a notifications source.</p>
 
                   <figure id="solid-quick-notifications-flow-receivefrom" rel="schema:hasPart" resource="#solid-quick-notifications-flow-receivefrom">
                     <img rel="schema:image" src="notifications-flow-receivefrom.mmd.svg" width="800" />
 
-                    <figcaption property="schema:name">Solid Quick Notifications Flow Receive From, where the receiver establishes connection to sender.</figcaption>
+                    <figcaption property="schema:name">Solid Quick Notifications Flow Receive From, where the receiver establishes a connection to a sender.</figcaption>
```

Comment:
Again, should match earlier. I think `a` is better than `the`, throughout.
```suggestion
                    <figcaption property="schema:name">Solid Quick Notifications Flow Receive From, where a receiver establishes a connection to a sender.</figcaption>
```